lib/cocoapods/target/pod_target.rb

```diff
@@ -244,34 +244,27 @@ def contains_test_specifications?
       !test_specs.empty?
     end
 
-    # @return [Hash{String=>Array<Hash{Symbol=>String}>}] The vendored and non vendored framework paths this target
+    # @return [Hash{String=>Array<Framework>}] The vendored and non vendored framework paths this target
     #         depends upon keyed by spec name. For the root spec and subspecs the framework path of the target itself
     #         is included.
     #
     def framework_paths
       @framework_paths ||= begin
         file_accessors.each_with_object({}) do |file_accessor, hash|
-          frameworks = []
-          file_accessor.vendored_dynamic_artifacts.map do |framework_path|
+          frameworks = file_accessor.vendored_dynamic_artifacts.map do |framework_path|
             relative_path_to_sandbox = framework_path.relative_path_from(sandbox.root)
-            framework = { :name => framework_path.basename.to_s,
-                          :input_path => "${PODS_ROOT}/#{relative_path_to_sandbox}",
-                          :output_path => "${TARGET_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/#{framework_path.basename}" }
+            framework_source = "${PODS_ROOT}/#{relative_path_to_sandbox}"
             # Until this can be configured, assume the dSYM file uses the file name as the framework.
             # See https://github.com/CocoaPods/CocoaPods/issues/1698
             dsym_name = "#{framework_path.basename}.dSYM"
             dsym_path = Pathname.new("#{framework_path.dirname}/#{dsym_name}")
-            if dsym_path.exist?
-              framework[:dsym_name] = dsym_name
-              framework[:dsym_input_path] = "${PODS_ROOT}/#{relative_path_to_sandbox}.dSYM"
-              framework[:dsym_output_path] = "${DWARF_DSYM_FOLDER_PATH}/#{dsym_name}"
```

Comment:
turns out `:dsym_name` & `:name` were unused, and `:output_path` and `:dsym_output_path` were only used to build the script output path list. so I pulled the logic for the output paths out of the target since it seems like it really shouldn't be there anyway